webapp/src/main/webapp/iaf/gui/js/controllers.js

```diff
@@ -1445,11 +1445,14 @@ angular.module('iaf.beheerconsole')
 		columnDefs: [ {
 			targets: 0,
 			render: function ( data, type, row ) {
-				if(type === 'display'){
-					for(let i in data){
+				if(type === 'display') {
+					data["messageId"] = data["id"];
+					for(let i in data) {
+						if(i == "id") continue;
 						var columnData = data[i];
-						if(typeof columnData == 'string' && columnData.length > 30)
-							data[i] = '<span class="ellipsis" title="'+columnData.replace(/"/g, '&quot;')+'">'+columnData.substr(0, 15)+' &#8230; '+columnData.substr(-15)+'</span>';
+						if(typeof columnData == 'string' && columnData.length > 30) {
+							data[i] = '<span title="'+columnData.replace(/"/g, '&quot;')+'">'+columnData.substr(0, 15)+' &#8230; '+columnData.substr(-15)+'</span>';
+						}
```

Comment:
so this 'modified' data is only to display and not process

Reply:
Okay, I see it now. Thanks!